A62-pick-first.md

```diff
@@ -0,0 +1,100 @@
+A62: `pick_first` LB policy
```

Comment:
I'd prefer not to present this as a complete spec for PF, because in the pending dualstack backend design (see pending https://github.com/grpc/proposal/pull/356, which will almost certainly be merged after this one), I am going to make some other significant changes to PF, and I don't want anyone to read this and think they have a complete picture of how PF works.

Instead let's structure it as describing the specific changes we're making here:
- sticky-TF behavior
- adding the shuffle option

The title can be something like "pick_first: sticky TRANSIENT_FAILURE and address order randomization".

We can still include a summary of the overall PF behavior in the background section, including a description of the sticky-TF behavior in C-core and how it works differently in Java/Go.

I think we should also explain *why* we're making these changes (without referencing our weird internal use-cases, because there's plenty of justification without that).  For the sticky-TF change, this includes:
- Avoids long delays before failing RPCs while the channel goes back to CONNECTING state while attempting to reconnect.
- Allows PF to work as a child of the priority policy (can reference [gRFC A56](https://github.com/grpc/proposal/blob/master/A56-priority-lb-policy.md)).

For the shuffle option, the justification is that there are some DNS configurations where all clients get the addresses in the same order (e.g., either because the authoritative server does that or because of caching) but where it is desirable to randomize the order of the addresses to provide better load balancing.

Reply:
Done. Thanks.